fenix/app/src/main/java/org/mozilla/fenix/library/bookmarks/DesktopFolders.kt

```diff
@@ -49,6 +49,10 @@ class DesktopFolders(
         }
     }
 
+    suspend fun count(): Int {
```

Comment:
This doesn't seem like it's *quite* the correct abstraction, but I can't see a better option and it seems close enough :)